packages/autorest.typescript/test/rlcIntegration/generated/multipleUrlParameters/package.json

```diff
@@ -42,21 +42,22 @@
     ]
   },
   "dependencies": {
-    "@azure-rest/core-client": "^1.4.0",
-    "@azure/core-auth": "^1.6.0",
-    "@azure/core-rest-pipeline": "^1.5.0",
-    "@azure/logger": "^1.0.0",
-    "tslib": "^2.6.2"
+    "@azure-rest/core-client": "^2.1.0",
```

Comment:
why we have this change?

Reply:
I see dev-tool below, so this should be for js repo and should have the workspace versions?